io-sim/src/Control/Monad/IOSim/STM.hs

```diff
@@ -171,12 +171,10 @@ writeTBQueueDefault (TBQueue queue _size) a = do
 
 isEmptyTBQueueDefault :: MonadSTM m => TBQueueDefault m a -> STM m Bool
 isEmptyTBQueueDefault (TBQueue queue _size) = do
-  (xs, _, ys, _) <- readTVar queue
+  (xs, _, _, _) <- readTVar queue
```

Comment:
It looks like your squash/rebase/etc is accidentally undoing PR #44, which merged 16 hours ago.

There was another merged about 4 hours ago. And then 3 days ago.

So you may want to double-check your new commit doesn't contain any other accidental changes

Reply:
This new commit still includes this change 
![image](https://user-images.githubusercontent.com/1106131/201121671-bdcda7d8-87fc-4fb9-aae0-ca735017b060.png)

Reply:
Fixed! 👍